packages/@aws-cdk/cfnspec/spec-source/000_CloudFormationResourceSpecification.json

```diff
@@ -60204,6 +60204,12 @@
       },
       "Documentation": "http://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.html",
       "Properties": {
+        "KmsKeyId": {
+          "Documentation": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-logs-loggroup.html#cfn-logs-loggroup-kmskeyid",
+          "PrimitiveType": "String",
+          "Required": false,
+          "UpdateType": "Mutable"
+        },
```

Comment:
spec changes are automated and not manually made -

the spec import is being done in #11319 ... let's wait for that to get merged and keep this change isolated to making the capability available

Reply:
Sounds good. Thanks :+1:

Reply:
@nicklaw5 it's been merged so updating your PR should allow us to simply consume the spec 🙂